README.md

```diff
@@ -776,6 +776,8 @@ scan:
                 [-ask-key-password]
                 [-debug]
                 [-pipe]
+                [-timeout]
+                [-timeout-detect-platform]
```

Comment:
I'm not sure about this suggestion, since we are in context of Scan, should be `-timeout` as main option related to it instead of platform detection?